packages/slate-editor/src/extensions/gallery-bookmark/GalleryBookmarkExtension.tsx

```diff
@@ -0,0 +1,7 @@
+import type { Extension } from '@prezly/slate-commons';
+
+export const EXTENSION_ID = 'GalleryBookmarkExtension';
+
+export const GalleryBookmarkExtension = (): Extension => ({
```

Comment:
The extension feels kinda useless here since it doesn't provide anything and we're using web bookmarks under the hood.

The only thing is the `createGalleryBookmark` function, but I was thinking if it would be easier to just place it inline to the `GalleryBookmarkPlaceholderElement` and get rid of this whole extension? 🤔